modules/storages/app/controllers/storages/admin/projects_storages_controller.rb

```diff
@@ -146,6 +146,13 @@ def destroy
     redirect_to project_settings_projects_storages_path
   end
 
+  def destroy_info
+    @project_storage_to_destroy = @object
+
+    hide_project_in_layout
```

Comment:
Are we sure to hide project's layout? Maybe, It makes sense not to, because we remove storage in the context of a project. Wdyt?

Reply:
Good point, I mostly followed the project delete example which actually shouldn't apply completely here. Great catch! I'll update 👍🏾